docs/Using-Pantheon/Events-and-Logs.md

```diff
@@ -0,0 +1,194 @@
+# Events and Logs
+
+When a transaction is mined, smart contracts emit events and write logs to the blockchain.  
+
+Logs are associated with the contract address and included in the blockchain but logs are not accessible 
```

Comment:
Logs are in separate storage to the blockchain, not directly in the blockchain - that's why log storage is cheaper than regular blockchain storage

Reply:
How about: Logs are associated with the contract address but are stored separately to the blockchain and logs are not accessible